cmd/manager/main.go

```diff
@@ -151,11 +155,23 @@ func newRootCommand() *cobra.Command {
 			hiveNSName := utils.GetHiveNamespace()
 			log.Infof("hive namespace: %s", hiveNSName)
 
+			// Create and start liveness and readyness probe endpoints
+			http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
+				w.WriteHeader(http.StatusOK)
+			})
+			http.HandleFunc("/readyz", func(w http.ResponseWriter, r *http.Request) {
+				if isLeader {
+					w.WriteHeader(http.StatusOK)
+				}
+				w.WriteHeader(http.StatusServiceUnavailable)
+			})
+			log.Info("Starting /healthz and /readyz endpoints")
+			go http.ListenAndServe(":8080", nil)
```

Comment:
I am not sure if this is the right way to do it but I tested it and works perfectly fine